src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -131,6 +132,28 @@ default Vector3i getHighestPositionAt(Vector3i position) {
     }
 
     /**
+     * Returns the y level that precipitation ends falling in the given column.
+     *
+     * <p>A value is still returned for columns in biomes which do not
+     * receive precipitation.</p>
+     *
+     * @return The y level that precipitation ends
+     */
+    int getPrecipitationLevel(int x, int z);
+
+    /**
+     * Returns the y level that precipitation ends falling in the given column.
+     *
+     * <p>A value is still returned for columns in biomes which do not
+     * receive precipitation.</p>
+     *
+     * @return The y level that precipitation ends
+     */
+    default int getPrecipitationLevel(Vector2i column) {
```

Comment:
Should there also be a `default Vector3i getPrecipitationLevelAt(Vector3i position)` method, for consistency with `getHighestYAt` and `getHighestPositionAt`? Similarly, if we're going to have this Vector2i method, should `getHighestYAt` also have a similar `Vector2i` method? It seems odd that we have two similar actions don't match up with each others' signatures.